pwa-devdocs/_drafts/pwa-studio-fundamentals/extensibility-with-targets/index.md

```diff
@@ -0,0 +1,543 @@
+---
+title: Modular Extensibility in PWA Studio with Targets
+---
+
+The new extensibility system for PWA Studio turns your storefront project into a self-organizing app, built by modules you can install, and hand-tuned by as much or as little customization code as you like.
+
+A few simple new concepts—the network of **Targets**, the **BuildBus**, and the [**Interceptor pattern**](https://web.archive.org/web/20170912094101/http://www.cs.wustl.edu/~schmidt/POSA/POSA2/access-patterns.html)—enable your chosen third-party code to enhance the build toolchain, add new functionality to the storefront, and even _rewrite the code of your published PWA on-the-fly_.
+
+# Quick Start
+
+Use Targets to add a new custom route to a Venia-based store, without editing any VeniaUI code.
+
+## Project Setup
+<!-- TODO replace with scaffolding setup ASAP -->
```

Comment:
I can handle this TODO

I was able to do this quickstart tutorial using this PR's scaffolding instead of a cloned repo